File: BE/capsule-service/src/main/java/com/timecapsule/capsuleservice/dto/OpenedCapsuleDto.java
package com.timecapsule.capsuleservice.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Builder;
import lombok.Getter;

import java.time.LocalDate;
import java.util.Date;

@Getter
public class OpenedCapsuleDto {
    private int capsuleId;
    private LocalDate openDate;
    private String address;
    @JsonProperty("isAdded")
    private boolean added;

    @Builder
    public OpenedCapsuleDto(int capsuleId, LocalDate openDate, String address, boolean isAdded) {
        this.capsuleId = capsuleId;
        this.openDate = openDate;
        this.address = address;
        this.added = isAdded;
    }
}

File: FE/AboutTimeCapsule/app/src/main/java/com/aboutcapsule/android/data/capsule/OpenedCapsuleDto.kt
package com.aboutcapsule.android.data.capsule

import com.google.gson.annotations.SerializedName
import java.time.LocalDate

data class OpenedCapsuleDto (

    @SerializedName("capsule_id")
    var capsuleId : Int ,

    @SerializedName("open_date")
    var openDate : String,

    var address : String ,

    @SerializedName("is_added")
    var isAdded : Boolean

)